src/drivers/interrupt/cortexm_irq_handle.S

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
The only reason for doing that is to keep this file along with cortexm_nvic.c, because it has only supporting magic for interrupt_handle realization and no its own separated logic.